routes/api.js

```diff
@@ -7,6 +7,14 @@ var debug = require('debug')('dashboard-proxy:server');
 var error = require('debug')('dashboard-proxy:server:error');
 var Buffer = require('buffer').Buffer;
 var BufferList = require('../node_modules/websocket/vendor/FastBufferList');
+var BufferList;
+try {
+    // from source tree
+    BufferList = require('../node_modules/websocket/vendor/FastBufferList');
+} catch(e) {
+    // from install location
+    BufferList = require('../../websocket/vendor/FastBufferList');
+}
```

Comment:
Yes, the path to that vendor asset is finicky depending on how node lays out the deps. I'll try it manually again to see if I can reproduce and guess another way to formulate the try/catch on that vendor path.

Sidenote: is there something better we could be doing here with this vendor bit?